engine/src/test/java/com/datatorrent/stram/client/AppPackageTest.java

```diff
@@ -95,19 +96,53 @@ public void testAppPackage() throws Exception
     JSONObject application1 = json.getJSONArray("applications").getJSONObject(0);
     JSONObject application2 = json.getJSONArray("applications").getJSONObject(1);
 
+
+
     Map<String, JSONObject> apps = new HashMap<>();
 
     apps.put(application1.getString("name"), application1);
     apps.put(application2.getString("name"), application2);
 
+
     Assert.assertEquals(true, apps.containsKey("MyFirstApplication"));
     Assert.assertEquals(true, apps.containsKey("MySecondApplication"));
 
-    Assert.assertEquals("mydtapp-1.0-SNAPSHOT.jar", apps.get("MyFirstApplication").getString("file"));
 
-    JSONObject dag = apps.get("MyFirstApplication").getJSONObject("dag");
-    Assert.assertTrue("There is at least one stream", dag.getJSONArray("streams").length() >= 1);
-    Assert.assertEquals("There are two operator", 2, dag.getJSONArray("operators").length());
+    JSONObject myFirstApplication = apps.get("MyFirstApplication");
+    JSONObject mySecondApplication = apps.get("MySecondApplication");
+    Assert.assertNotNull("MyFirstApplication not found", myFirstApplication);
```

Comment:
This might not be needed as application object can not be null as we are setting non-null object only in map